homeassistant/components/homewizard/coordinator.py

```diff
@@ -74,9 +74,10 @@ async def _async_update_data(self) -> DeviceResponseEntry:
 
                 # Do not reload when performing first refresh
                 if self.data is not None:
-                    await self.hass.config_entries.async_reload(
+                    self.hass.config_entries.async_schedule_reload(
                         self.config_entry.entry_id
                     )
+                    )
```

Comment:
Looks like I messed up my previous suggestion
```suggestion
```

Reply:
Can happen! Updated this locally and your suggestion works as expected. Now working to see how on 🌍 I can add tests 😇